counterfit/frameworks/art/art.py

```diff
@@ -448,34 +190,66 @@ def run(self, cfattack: CFAttack):
         else:
             print("Not found!")
         return results
-
-    def post_attack_processing(self, cfattack: CFAttack):
-        attack_attributes = set(cfattack.attack.__dir__())
-
-        if "generate" in attack_attributes:
-            current_datatype = cfattack.target.target_data_type
-            current_dt_report_gen = get_target_data_type_obj(current_datatype)
-            summary = current_dt_report_gen.get_run_summary(cfattack)
-            current_dt_report_gen.print_run_summary(summary)
+    
+    @classmethod
+    def pre_attack_processing(cls, cfattack: CFAttack):
+        cls.set_parameters(cfattack)
+
+    @staticmethod
+    def post_attack_processing(cfattack: CFAttack):
+        attack_attributes = cfattack.attack.__dir__()
+
+        pass
+
+        # if "generate" in attack_attributes:
+        #     current_datatype = cfattack.target.data_type
+        #     current_dt_report_gen = get_target_data_type_obj(current_datatype)
+        #     cfattack.summary = current_dt_report_gen.get_run_summary(cfattack)
+        #     # current_dt_report_gen.print_run_summary(summary)
             
-        elif "extract" in attack_attributes:
-            # Override default reporting for the attack type
-            extract_table = Table(header_style="bold magenta")
-            # Add columns to extraction table
-            extract_table.add_column("Success")
-            extract_table.add_column("Copy Cat Accuracy")
-            extract_table.add_column("Elapsed time")
-            extract_table.add_column("Total Queries")
-
-            # Add data to extraction table
-            success = cfattack.success[0]  # Starting value
-            thieved_accuracy = cfattack.results
-            elapsed_time = cfattack.elapsed_time
-            num_queries = cfattack.logger.num_queries
-            extract_table.add_row(str(success), str(
-                thieved_accuracy), str(elapsed_time), str(num_queries))
-
-            CFPrint.output(extract_table)
+        # elif "extract" in attack_attributes:
+        #     # Override default reporting for the attack type
+        #     extract_table = Table(header_style="bold magenta")
+        #     # Add columns to extraction table
+        #     extract_table.add_column("Success")
+        #     extract_table.add_column("Copy Cat Accuracy")
+        #     extract_table.add_column("Elapsed time")
+        #     extract_table.add_column("Total Queries").attack_id: str = uuid.uuid4(
+
+        #     # Add data to extraction table
+        #     success = cfattack.success[0]  # Starting value
+        #     thieved_accuracy = cfattack.results
+        #     elapsed_time = cfattack.elapsed_time
+        #     num_queries = cfattack.logger.num_queries
+        #     extract_table.add_row(str(success), str(
+        #         thieved_accuracy), str(elapsed_time), str(num_queries))
+
+        #     CFPrint.output(extract_table)
```

Comment:
Could you clean the commented code and remove `pass` statement.

Reply:
Fixed in a5c7a40738048d0b5d977e218edfc428d729a233